### DISCUSSION TITLE
How to avoid a table splitting in between rows

### DISCUSSION THREAD

**Discussion by PatriciaFLJacob** [dolanmiu/docx#991] [Q&A]
I have a document that can greatly vary in length, which means our tables can often be end up at the end of a page. I would like to keep them from ever splitting across two pages. I have tried a combination of keepNext on the Paragraph of the first cell of each row (as well as in all of them), and setting cantSplit to each TableRow, but it is still getting divided. Is there a way around this? Or might I suggest adding some sort of keepNext feature to TableRow the same as we have for Paragraph?

Thanks for any suggestions/help!

**Comment by protobi-pieter** [comment]
Have any developers following this repo added a feature like this or are willing do so?  We'd be interested in sponsoring features that  keep table rows and tables from unnecessarily breaking across pages, and when necessary, repeating column headings on each page.
These are possible in the MS Word application itself, so should be possible to add here.   I could get by with a working hack on a separate fork, and from there would prefer get that to a quality level where it is an accepted PR on the main branch.
